robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -178,6 +183,14 @@ public void run() {
       }
     });
 
+    ViewRootImpl root = component.getWindow().getDecorView().getViewRootImpl();
+    Display display = Shadow.newInstanceOf(Display.class);
```

Comment:
In the Robolectric tests this seems like the way to do it.

https://github.com/robolectric/robolectric/search?utf8=%E2%9C%93&q=shadowdisplay